src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+const localeCompare = function(a, b) {
+  const firstString = String(a);
+  const secondString = String(b);
+
+  if (firstString > secondString) {
+    return 1;
+  } else if (firstString === secondString) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = localeCompare) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => String(a) > String(b) {
```

Reply:
Your solution is good, but I want you to know that it will give the same result

Reply:
Thank you very much! I haven`t considered comparing them with ">" sign.